packages/nginx.rb

```diff
@@ -16,14 +16,28 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    system "sudo ln -sf /usr/local/nginx/sbin/nginx /usr/local/bin/nginx"
-    system "echo all NGINX things are in /usr/local/nginx"
-    system "echo pages are in /usr/local/nginx/html"
-    system "echo adding bash aliases so you can easily start/stop nginx"
-    system "echo startnginx starts nginx and stopnginx stops nginx"
-    system "sed -i \'/^alias startnginx/d\' ~/.bashrc"
-    system "sed -i \'/^alias stopnginx/d\' ~/.bashrc"
-    system "sed -i '$ a alias startnginx=\"sudo nginx\"' ~/.bashrc"
-    system "sed -i '$ a alias stopnginx=\"sudo nginx -s quit\"' ~/.bashrc"
+    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/bin"
+    FileUtils.cd("#{CREW_DEST_DIR}#{CREW_PREFIX}/bin") do
+      system "sudo ln -sf #{CREW_PREFIX}/nginx/sbin/nginx nginx"
```

Comment:
Please remove this sudo.  It works without sudo since making symbolic link under #{CREW_DEST_DIR}.
It is also possible to move `system "echo..."` commands here.